_rules/SC1-1-1-image-has-name.md

```diff
@@ -19,20 +19,18 @@ authors:
 
 ### Applicability
 
-The rule applies to any HTML element with the [semantic role][] of `image` that is [exposed to assistive technologies][].
+The rule applies to any HTML element with the [semantic role](#semantic-role) of `image` that is [exposed to assistive technologies](#exposed-to-assistive-technologies).
 
-**NOTE**: Any image marked as decorative, by using either `role="presentation"`, `role="none"` or an empty `alt` attribute in combination with either no `title` attribute or an empty `title` attribute, does not have the role of `image` according to [Accessible Rich Internet Applications (WAI-ARIA) 1.1](https://www.w3.org/TR/wai-aria-1.1/)
+**NOTE**: Any image marked as decorative, by using either `role="presentation"`, `role="none"` or an empty `alt` attribute, does not have the role of `image` according to [ARIA in HTML](https://www.w3.org/TR/html-aria/#img-alt).
 
 ### Expectation
 
-Each target element has an [accessible name][] that is [non-empty][].
-
-## Assumptions
```

Comment:
Why did the assumptions section get removed? Clearly there is an assumption here that role=img is only given to actual images.

Reply:
What if we changed the applicability and expectation to the following...
Applicability
The rule applies to HTML image elements or any HTML element with the semantic role of image that is exposed to assistive technologies.

Expectation
Each target element has an accessible name that is non-empty or is marked as decorative.

NOTE: An image can be marked as decorative, by using either role="presentation", role="none" or an empty alt attribute alt="".